inference-engine/src/transformations/src/transformations/hsigmoid_fusion.cpp

```diff
@@ -0,0 +1,210 @@
+// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
We need also a decomposition transformation for HSigmoid for the plugins which do not support HSigmoid directly. For now it must be enabled for all plugins.